website/docs/d/ess_scaling_configurations.html.markdown

```diff
@@ -50,6 +50,7 @@ The following attributes are exported in addition to the arguments listed above:
   * `internet_charge_type` - Internet charge type of the scaling configuration.
   * `internet_max_bandwidth_in` - Internet max bandwidth in of the scaling configuration.
   * `internet_max_bandwidth_out` - Internet max bandwidth of the scaling configuration.
+  * `credit_specification` - Performance mode of the t5 burstable instance.
```

Comment:
This is datasource output and if you want to support it, it should be added in its data source code.

Reply:
Fair enough. Added to datasources now